<ui version="4.0" >
 <class>frm3dClass</class>
 <widget class="QWidget" name="frm3dClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1255</width>
    <height>954</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>frm3d</string>
  </property>
  <widget class="QPushButton" name="pushButton" >
   <property name="geometry" >
    <rect>
     <x>1170</x>
     <y>920</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text" >
    <string>Close(&amp;C)</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>920</y>
     <width>168</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" >
    <item>
     <widget class="QLabel" name="label" >
      <property name="text" >
       <string>PageIndex</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="txtPageIdx" />
    </item>
   </layout>
  </widget>
  <widget class="QFrame" name="frame" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>10</y>
     <width>1231</width>
     <height>901</height>
    </rect>
   </property>
   <property name="frameShape" >
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow" >
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_2" >
   <property name="geometry" >
    <rect>
     <x>450</x>
     <y>920</y>
     <width>215</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" >
    <item>
     <widget class="QLabel" name="label_3" >
      <property name="text" >
       <string>Y AXISLine</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="cmbLine_2" />
    </item>
    <item>
     <widget class="QPushButton" name="btnLine_2" >
      <property name="text" >
       <string>X LineCSV</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget" >
   <property name="geometry" >
    <rect>
     <x>221</x>
     <y>920</y>
     <width>215</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" >
    <item>
     <widget class="QLabel" name="label_2" >
      <property name="text" >
       <string>X AXISLine</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="cmbLine_1" />
    </item>
    <item>
     <widget class="QPushButton" name="btnLine_1" >
      <property name="text" >
       <string>Y LineCSV</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="" >
   <property name="geometry" >
    <rect>
     <x>701</x>
     <y>920</y>
     <width>242</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout" >
    <item>
     <widget class="QLabel" name="label_4" >
      <property name="text" >
       <string>Average</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="avgx" >
      <property name="value" >
       <number>3</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_5" >
      <property name="text" >
       <string>Ã—</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="avgy" >
      <property name="value" >
       <number>3</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_2" >
      <property name="text" >
       <string>AverageView</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources/>
 <connections/>
</ui>
